travis/install_custom_pg

```diff
@@ -7,25 +7,31 @@ if [ -z "${USE_CUSTOM_PG:-}" ]; then
   exit
 fi
 
-# in order for other PostgreSQL packages to operate correctly, we need
-# to ensure our build uses the same directories as the PGDG build; un-
-# fortunately, not all can be specified by ./configure, so we apply the
-# version-specific packaging patch used for the PGDG builds
-pkgingurl='https://anonscm.debian.org/git/pkg-postgresql/postgresql.git'
-patchurl="${pkgingurl}/plain/debian/patches/50-per-version-dirs.patch"
-
-# clone PostgreSQL
+# if there is cached PostgreSQL build, just pull new updates, hopefuly
+# there are no updates and we can skip the compiling PostgreSQL
 cd ~
-git clone -b "REL${PGVERSION//./_}_STABLE" --depth 1 git://git.postgresql.org/git/postgresql.git
+if test "$(ls -A postgresql)"; then
```

Comment:
This will produce an error message when the directory does not exist (most of the time). Maybe use `if [ -d "postgresql" ]`.